Unix/doc/setup-kerberos-omi.md

```diff
@@ -37,7 +37,15 @@ Table of Contents:
     - [Setting up Credentials](#setting-up-credentials)
 - [Troubleshooting Authentication](#troubleshooting-authentication)
     - [Common Problems](#common-problems)
-        - [Troubleshooting Steps](#troubleshooting-steps)
+        - [Cause 1: Wrong domain/user/password on command line](#common-problems)
+        - [Cause 2: Expired Credentials](#common-problems)
+        - [Cause 3: Time out of sync with host](#common-problems)
+        - [Cause 4: SSSD/ Winbind setup](#common-problems)
+        - [Cause 5: KDC server dns issue](#common-problems)
+        - [Cause 6: Lower domain name](#common-problems)
+        - [Cause 7: knvo inconsistent issue](#common-problems)
+        - [Cause 8: keytab setting issue](#common-problems)
```

Comment:
For all of these causes, please jump EXACTLY to the entry in question, and not to the #common-problems section, thanks.